Bluepill-cli/Bluepill-cli/Simulator/SimulatorRunner.m

```diff
@@ -61,6 +61,27 @@ - (void)createSimulatorWithDeviceName:(NSString *)deviceName completion:(void (^
     }];
 }
 
+- (BOOL)useSimulatorWithDeviceUDID:(NSUUID *)deviceUDID {
+    self.device = [SimulatorRunner findDeviceWithConfig:self.config andDeviceID:deviceUDID];
+    if (self.device) {
```

Comment:
There are. But the code block are small, so the readability are pretty good. Any better way?